src/commands/app/logs.ts

```diff
@@ -128,6 +127,10 @@ export default class AppLogs extends Command {
         lastLogUnix = parseInt(unixTime) + 1;
       }
       await this.sleep(1000);
+      if (start) {
+        start += 1;
+        this.debug('since timpestamp', start);
```

Comment:
```suggestion
        debug && this.debug('since timpestamp', start);
```

We should only output if debug flag is given.

Reply:
It already works as expected but alright.